contributingGuides/STYLE.md

```diff
@@ -540,96 +482,48 @@ const propTypes = {
 }
 ```
 
-## Binding methods
```

Comment:
This is class specific and totally unnecessary in function components.

Reply:
I think I would come back and update this later as part of some evolving best practices. My instinct is to say something like...

If you aren't going to memoize the function with `useCallback()` then make it a pure function and move it outside the component if you can. If you need to create a closure around the props or state or something else then either allow it to be redeclared or just `useCallback()`. Probably the community has some idea about what would be best.